README.md

```diff
@@ -57,6 +57,17 @@ The event method has the following API:
 * `tags`:       The Array of tags to add to metrics `default: []`
 * `callback`:   The callback to execute once the metric has been sent.
 
+The check method has the following API:
+
+* `name`:        Check name `required`
+* `status`:      Check status `required`
+* `options`:     Options for the check
+  * `date_happened`    Assign a timestamp to the check `default is now`
+  * `hostname`         Assign a hostname to the check.
+  * `message`          Assign a message to the check.
+* `tags`:       The Array of tags to add to metrics `default: []`
+* `callback`:   The callback to execute once the metric has been sent.
```

Comment:
Can you add a super-simple example to the section right below this and mention the check method in the "DogStatsD and Telegraf functionality" section?  Thanks!